src/admin/components/Button.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from "classnames";
+import {
+  AnchorHTMLAttributes,
+  ButtonHTMLAttributes,
+  InputHTMLAttributes,
+} from "react";
+
+import styles from "./Button.module.scss";
+
+export function Button({
+  className,
+  children,
+  ...rest
+}: ButtonHTMLAttributes<HTMLButtonElement>) {
+  return (
+    <button className={classNames(styles.button, className)} {...rest}>
+      {children}
+    </button>
+  );
+}
+
+export function Input({
+  className,
+  children,
+  ...rest
+}: InputHTMLAttributes<HTMLInputElement>) {
+  return (
+    <input className={classNames(styles.button, className)} {...rest}>
+      {children}
+    </input>
+  );
+}
+
+export function A({
```

Comment:
Can we find a better name than A?

Reply:
Maybe for all of the components in here, as Button, Input and A all represent native HTML tags.
In Vue, there is a guideline of always choosing a component name with at least 2 words.

Maybe something like BrandButton, BrandInput, BrandA or BrandAnchor.